### ISSUE TITLE
RFE: Lifepath section standardization

### ISSUE THREAD

**Issue by SirChaos1** [MegaMek/mekhq#4513]
In the Lifepath section of the campaign options, the odds for random marriage, divorce and procreation are given as percent per day, whereas the chance of dropout/washout and accidents for the education module are given as die sizes and rolled weekly.

Wouldn´t it make more sense to change random marriage, divorce and procreation chances to be set as die sizes (i.e. die size 1,000 instead of a 0.1 percent chance) as well? I think die size, showing the 1-in-X chance of something happening, is more intuitive; knowing something happens once every X days on average makes it easier to estimate how often it will happen over the course of Y years, whereas showing a percentage chance adds one per math step to the estimate.

Also, since accidents and dropouts only happening on Monday has worked well enough so far, I think it would be worth considering only rolling for random marriage, divorce and procreation once per week (Sunday, maybe, to spread the events around the week) with proportionately higher odds instead of daily, thus presumably reducing daily workload in larger units.

**Comment by IllianiCBT** [comment]
This has been actioned as part of an overall review of Procreation, Marriage, and Divorce chances - the originals, while functional, produced some odd behavior.

At time of writing Procreation has been done (took changes across 12 separate files - which surprised me), but I'm going to leave this case open to foster discussion on what the new chances should be.

Currently I have random procreation dice size as `612` (in relationship) or `1861` (out of relationship). This is factoring in likelihood of pregnancy, success rate of condom usage, and increased likelihood of intercourse occurring. All based on real world data. The dice are rolled once a week (I prefer weekly checks as it reduces overall load by about 85% compared to daily checks) and pregnancy occurs on a roll of `1`.

I've also added a control: the dice size increases by a multiple equal to the number of current children (minimum of 1). So the chance of having a child decreases exponentially the more children currently 'owned'. This should help reduce the Mormon-sized families we see occurring with the current system.

I opted not to simulate age-based modifiers as I felt a degree of abstraction was better than overburdening the simulation with too many modifiers (reducing user 'readability' of the mechanic). Though I am open to alternative suggestions.
